gpytorch/gpytorchopt/gpytorchopt/acquisition/functional/batch_acquisition.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+from gpytorch import Module
+from math import pi, sqrt
+import torch
+import torch.nn.functional as F
+
+from math import inf
+
+
+def batch_simple_regret(
+    x: torch.Tensor,
+    model: Module,
+    mc_samples: int=1000,
+) -> torch.Tensor:
+    # let's be paranoid
+    x.requires_grad = True
+    model.eval()
+    val = model.forward(x).sample(mc_samples).max(0)[0].mean()
```

Comment:
Calling forward manually instead of using `__call__` will get you the prior mean and covariance instead of the posterior. I haven't read the paper yet -- is this intended? Or should it be `model(x)`

Reply:
it should def. be the posterior